src/test/account.test.js

```diff
@@ -202,7 +201,8 @@ describe("Get Account by ID", () => {
       .expect("Content-Type", /application\/json/)
       .expect(400);
   });
-  test("If the ID has a correct format but there is any scope saved with that ID should return a 400", async () => {
+
+  test("If the ID has a correct format but there is any account saved with that ID should return a 400", async () => {
     const token = await databaseConnection();
```

Comment:
Why you are calling twice to get a token? 206 and 222 code lines

Reply:
are two test differents.